code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -167,3 +168,23 @@ proc/alien_type_present(var/alienpath)
 			continue
 		return 1
 	return 0
+
+//When the alien queen dies, all aliens suffer a penalty as punishment for failing to protect her.
+/mob/living/carbon/alien/humanoid/proc/queen_death()
+	if(stat == DEAD)
+		return
+	src << "<span class='userdanger'>Your queen has been struck down!</span>"

```

Comment:
"Your queen has perished!" May be a better, more universal term. 'Struck down' doesn't quite fit if, for example, an accidental plasma fire killed the queen.